### ISSUE TITLE
Video is hard to follow with release of Bootstrap v4 

### ISSUE THREAD

**Issue by fuzzylita** [learn-co-curriculum/bootstrap-navbar-code-along#5]
Hi! 

The instructions are designed to follow along with bootstrap v3 (I guess) but with the release of bootstrap v4 last week, they're a little out of sync. Some of the elements don't work with the older styling (navbar) and it's hard to figure out how to orient the social links to the right side of the bar.

Might be worth clarifying in the text! Hope that's helpful.

**Comment by jenmyers** [comment]
Thanks for reporting this issue. We're aware of it and are tracking it in #12. We're closing this specific issue but you can follow progress at the other URL.

-- Flatiron Curriculum Team
